internal/locale/errors.go

```diff
@@ -27,8 +29,8 @@ func (e *LocalizedError) Error() string {
 	return e.localized
 }
 
-// UserError is the user facing error message, it's the same as Error() but identifies it as being user facing
-func (e *LocalizedError) LocalizedError() string {
+// LocaleError is the user facing error message, it's the same as Error() but identifies it as being user facing
+func (e *LocalizedError) LocaleError() string {
```

Comment:
I was scratching my head for a while on this PR, debugging it nothing seemed wrong. Seeing no other options I figured I'd try just renaming the symbol as that's all that was done in the other PR that affected the behaviour, and in fact that did fix the issue.

I have NO idea why this fixes it though :\ Or rather, why it breaks with the name being `LocalizedError`.